common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -363,7 +364,11 @@ def create_user_for_course(self, course, user_type=CourseUserType.ENROLLED):
 
 
 class SharedModuleStoreTestCase(
-    ModuleStoreTestUsersMixin, FilteredQueryCountMixin, ModuleStoreIsolationMixin, CacheIsolationTestCase
+    ModuleStoreTestUsersMixin,
+    FilteredQueryCountMixin,
+    ModuleStoreIsolationMixin,
+    CacheIsolationTestCase,
+    OpenEdxEventsTestCase,
```

Comment:
I would say we can start placing this in the individual test cases. Only after the dust settles we can see if we want to have an impact such as this.